scss/bitstyles/atoms/card/_settings.scss

```diff
@@ -1,14 +1,19 @@
 @use '../../tools/palette';
+@use '../../tools/shadow';
 @use '../../tools/size';
 
 $sizes: (
   'base': size.get('m'),
   'l': size.get('xl'),
 ) !default;
 $border-radius: size.get('m') !default;
-$box-shadow:
-  0 size.get('xs') size.get('m') rgba(palette.get('brand-1', '100'), 0.1),
-  0 size.get('xxxs') size.get('xxs') rgba(palette.get('gray', '90'), 0.04),
-  0 0 size.get('xxs') rgba(palette.get('gray', '90'), 0.1) !default;
+$box-shadow: shadow.to-css(
+  shadow.generate(
+    $color: rgba(palette.get('brand-1', '100'), 0.025),
+    $offset: size.get('s'),
+    $blur-radius: size.get('l'),
+    $layers: 6,
```

Comment:
Yes this was done visually, but I’ll make another pass over all these changed shadows in storybook, to double-check they’re close in appearance 👍 

A general note about these changed shadows: because they’re built from multiple shadows, they are darker/heavier than the old single-layer shadows, and therefore the colors need to be more transparent.

This one specifically, switched to using 6 instead of 3 layers (all the other components used only single layer iirc) so they see some improvement similar to the other shadows, and because it’s now very easy to generate these shadows

Reply:
I just checked through all the shadows, and tweaked the button & dropdown shadows 🎨